tests/trainer/test_checkpoint.py

```diff
@@ -373,10 +381,19 @@ def test_surgery_resumption(self, tmp_path: pathlib.Path):
 
 class TestCheckpointResumption:
 
-    def get_trainer(self, **kwargs):
+    def get_trainer(self, checkpoint_save_interval=None, checkpoint_save_path=None, checkpoint_filename=None, **kwargs):
         model = SimpleConvModel()
         optimizer = torch.optim.Adam(model.parameters())
 
+        checkpoint_saver_kwargs = {}
+        if checkpoint_filename:
+            checkpoint_saver_kwargs.update({'checkpoint_filename': checkpoint_filename})
+        if checkpoint_save_path:
+            checkpoint_saver_kwargs.update({'checkpoint_save_path': checkpoint_save_path})
+        if checkpoint_save_interval:
+            checkpoint_saver_kwargs.update({'checkpoint_save_interval': checkpoint_save_interval})
+
+        checkpoint_saver = CheckpointSaver(**checkpoint_saver_kwargs)
```

Comment:
same comment here as above, this is important for readability of the tests, that the checkpointsaver creation is actually being done by the test, not the helper function.

Reply:
done